src/k8s-engine/lima.ts

```diff
@@ -177,6 +177,12 @@ const IMAGE_VERSION = '0.2.15';
 const ALPINE_EDITION = 'rd';
 const ALPINE_VERSION = '3.15.4';
 
+const CREDFWD_DIR = '/etc/rancher/desktop';
```

Comment:
Please have better names / comments for these constants (since they're now far away from `installCredentialHelper()`).

Reply:
Makes sense.